src/Riok.Mapperly/Descriptors/SymbolAccessor.cs

```diff
@@ -16,6 +16,10 @@ public class SymbolAccessor
     private readonly Dictionary<ITypeSymbol, IReadOnlyCollection<ISymbol>> _allMembers = new(SymbolEqualityComparer.Default);
     private readonly Dictionary<ITypeSymbol, IReadOnlyCollection<IMappableMember>> _allAccessibleMembers =
         new(SymbolEqualityComparer.Default);
+    private readonly Dictionary<ITypeSymbol, IReadOnlyDictionary<string, IMappableMember>> _allAccessibleMembersCaseInsensitive =
```

Comment:
Aren't these dictionary always empty?

Reply:
Nope, they are filled on L199: `membersBySymbol[symbol] = symbolMembers = GetAllAccessibleMappableMembers(symbol)...` with `membersBySymbol` beeing one of the two (L193 `var membersBySymbol = ignoreCase ? _allAccessibleMembersCaseInsensitive : _allAccessibleMembersCaseSensitive;`).

Reply:
Missed that.